async_fuse/src/byte_slice.rs

```diff
@@ -53,10 +61,12 @@ impl<'a> ByteSlice<'a> {
         Ok(bytes)
     }
 
-    pub(crate) fn fetch<T: FuseAbiData>(&mut self) -> anyhow::Result<&'a T> {
+    /// Fetch some bytes and build target instance
+    pub fn fetch<T: FuseAbiData>(&mut self) -> anyhow::Result<&'a T> {
         let elem_len: usize = mem::size_of::<T>();
 
-        if (self.data.as_ptr() as usize) % mem::align_of::<T>() != 0 {
+        let address = unsafe { mem::transmute::<*const u8, usize>(self.data.as_ptr()) };
+        if address.overflow_rem(mem::align_of::<T>()) != 0 {
```

Comment:
The value of `mem::align_of::<T>()` is always a power of two and positive. 
So the unsigned integer division here is impossible to overflow.
Consider using  `wrapping_rem`.

See <https://doc.rust-lang.org/stable/reference/glossary.html#alignment>

Reply:
same above